loc/templates/page.html

```diff
@@ -19,7 +19,9 @@
 
     <noscript><!-- without javascript the pageviewer does not work; display OCR in this case instead. -->
         <h3>Newspaper Page Text</h3>
-        <pre>{{ page.ocr.text }}</pre>
+        {% if text %}
+        <pre>{{ text.0|linebreaks }} </pre>
```

Comment:
Is this really just text.0 or is it possible to have more than one line, where we'd need something like `|join`?

Reply:
added join filter to the template